src/seedu/addressbook/Main.java

```diff
@@ -58,13 +60,17 @@ private void start(String[] launchArgs) {
         } catch (InvalidStorageFilePathException | StorageOperationException e) {
             ui.showInitFailedMessage();
             /*
-             * ==============NOTE TO STUDENTS=========================================================================
-             * We are throwing a RuntimeException which is an 'unchecked' exception. Unchecked exceptions do not need
-             * to be declared in the method signature.
-             * The reason we are using an unchecked exception here is because the caller cannot reasonably be expected
-             * to recover from an exception.
-             * Cf https://docs.oracle.com/javase/tutorial/essential/exceptions/runtime.html
-             * =======================================================================================================
+             * ==============NOTE TO
+             * STUDENTS=========================================================
```

Comment:
Note this sometimes happens when using the auto indent feature of eclipse.
Do be careful.